src/logic/calculate.js

```diff
@@ -72,8 +73,7 @@ const operationButton = (data, buttonName) => {
       operation = buttonName;
     }
   }
-
   return { total, next, operation };
 };
-const calculate = (data, buttonName) => operationButton(data, buttonName);
+// const calculate = (data, buttonName) => operationButton(data, buttonName);
```

Comment:
It’s better to delete commented code. If you need to review a previous version of your code, use git history.